lib/assets/javascripts/_modules/search.js

```diff
@@ -50,12 +52,16 @@
       // Search functionality on search text input
       $searchInput.on('input', function (e) {
         e.preventDefault();
-        var query = $(this).val();
+        query = $(this).val();
```

Comment:
Do we need to do any sanitisation on this before sending it to GA? We do scrub things that look like PII from data sent to GA in other places, eg https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/analytics/analytics.js

I'm not aware of any uses where I'd expect users to type in PII but as this is a template it's hard to foresee what it will be used for.

I would suggest either making this functionality opt-in, or adding some explanation of what it does to the [ga-tracking-id](https://alphagov.github.io/tech-docs-manual/#ga-tracking-id) documentation, so that site creators are aware of this.

Reply:
That's a good idea. I've borrowed that stripPII function. I'll leave the full integration for #34.

I'll talk to @jonathanglassman tomorrow about making sure all the new GA tracking events are documented. 

I've also opened an issue about the date regex: alphagov/govuk_frontend_toolkit#469